src/collection-view.js

```diff
@@ -711,9 +712,14 @@ const CollectionView = Backbone.View.extend({
     return false;
   },
 
+  // Override to handle DOM inserting differently
+  beforeEl(el, siblings) {
+    this.Dom.getEl(el).before(siblings);
```

Comment:
As is now a native implementation for getEl would break this part of the code. I'd be happy with any solution that allows a native implementation